Update README.md

Valentina-Raffreddato-JCMaxwell-5BI opened the pull request.

Valentina-Raffreddato-JCMaxwell-5BI committed:
Update README.md

Luca-Volturno-JCMaxwell-5BI commented:
C'è qualche imprecisione ovvero, è necessario prima andare nel sito di Java e scaricare il pacchetto JDK perchè altrimenti nel prompt non si può eseguire il seguente comando: java 

Inoltre lo scopo del progetto non è: Inviare un messaggio attraverso il prompt 
Ma: Effettuare una connessione client/server con l'utilizzo di Sockets 

Poi c'è qualche imprecisione grammaticale: "+Una volta scaricato e installato dovete entrarci " 
Dovreste scrivere: Una volta scaricato e installato, si deve aprire

Poi anche questo: "apparirà una seconda finestra"
Sostituirlo cosi: "Verrà mostrata una seconda finestra con i vari tipi di file di Java

Andrea-Pappalardo-JCMaxwell-5Bi committed:
Update README.md

Luca-Volturno-JCMaxwell-5BI committed:
Aggiornamento readme -

Luca-Volturno-JCMaxwell-5BI committed:
Aggiornamento readme - Correzione errori grammaticali

Ho modificato qualche riga per modificare al meglio la presentazione della guida, in modo tale che qualsiasi utente possa capire immediatamente lo scopo del progetto

Andrea-Pappalardo-JCMaxwell-5Bi committed:
Update README.md

Andrea-Pappalardo-JCMaxwell-5Bi committed:
Update README.md

Andrea-Pappalardo-JCMaxwell-5Bi committed:
Update README.md

Luca-Volturno-JCMaxwell-5BI committed:
Merge pull request #13 from Valentina-Raffreddato-JCMaxwell-5BI/Andrea-Pappalardo-JCMaxwell-5Bi-patch-4

Update README.md

Luca-Volturno-JCMaxwell-5BI committed:
Merge pull request #12 from Valentina-Raffreddato-JCMaxwell-5BI/Andrea-Pappalardo-JCMaxwell-5Bi-patch-3

Update README.md

Luca-Volturno-JCMaxwell-5BI committed:
Merge pull request #11 from Valentina-Raffreddato-JCMaxwell-5BI/Andrea-Pappalardo-JCMaxwell-5Bi-patch-2

Update README.md

Luca-Volturno-JCMaxwell-5BI committed:
Merge pull request #10 from Valentina-Raffreddato-JCMaxwell-5BI/Andrea-Pappalardo-JCMaxwell-5Bi-patch-1

Update README.md
